Core/Core/Keychain/Keychain.swift

```diff
@@ -54,6 +54,17 @@ public struct KeychainEntry: Codable, Hashable {
         return masquerader?.host.flatMap { URL(string: "https://\($0)") }
     }
 
+    public var uniqueID: String {
```

Comment:
This feels a lot like a hashValue.  Have you considered just using `hashValue` of the KeychainEntry?

Reply:
I thought about using `hashValue` but `Hashable` isn't guaranteed to be the same across executions because they use a random seed each time. Docs say this:

```
    /// Hash values are not guaranteed to be equal across different executions of
    /// your program. Do not save hash values to use during a future execution.
```

That makes me think it's not a great fit for UserDefaults since they are persisted across executions.